src/main/java/RacingGame/controller/RacingGameController.java

```diff
@@ -14,9 +14,15 @@ public RacingGameController(MovableStrategy movableStrategy) {
 
     public void play() {
         String carNames = InputView.InputCarNames();
-        int StageCount = InputView.InputStageCount();
+        int stageCount = InputView.InputStageCount();
 
         ResultView.viewResultPhrase();
-        ResultView.viewResult(racingGameService.play(carNames, StageCount));
+        racingGameService.init(carNames);
+
+        for (int i = 0; i < stageCount; i++) {
+            ResultView.stageStatus(racingGameService.play());
+        }
+
+        ResultView.showWinners(racingGameService.winners());
```

Comment:
앗 컨트롤러에 비즈니스 로직이 들어갔네요!. 서비스 계층에 좀 더 책임을 줘도 될 것 같아요!
```suggestion
        RacingResult racingResult = racingGameService.play();
        
        ResultView.showPrgress(racingResult.getHistories());
        RacingView.showWinners(racingResult.winners());
```